cinema/tests/test_movie_session_api.py

```diff
@@ -46,9 +46,7 @@ def test_get_movie_sessions(self):
         }
         self.assertEqual(movie_sessions.status_code, status.HTTP_200_OK)
         for field in movie_session:
-            self.assertEqual(
-                movie_sessions.data[0][field], movie_session[field]
-            )
+            self.assertEqual(movie_sessions.data[0][field], movie_session[field])
```

Comment:
Ensure that the API response includes the fields 'movie_title', 'cinema_hall_name', and 'cinema_hall_capacity' as expected in the test. If the response structure differs, adjust the test to match the actual response.